typescript/nomad-sdk/src/nomad/govern/index.ts

```diff
@@ -61,73 +91,77 @@ export class CallBatch {
     overrides?: ethers.Overrides,
   ): Promise<ethers.PopulatedTransaction> {
     if (this.built) return this.built;
-
     const [domains, remoteCalls] = utils.associateRemotes(this.remote);
-
     this.built =
-      await this.core.governanceRouter.populateTransaction.executeGovernanceActions(
+      await this.governorCore.governanceRouter.populateTransaction.executeGovernanceActions(
         this.local,
         domains,
         remoteCalls,
       );
-
     return this.built;
   }
 
+  // Return the batch hash for the specified domain
+  domainHash(domain: number): string {
+    const calls = this.remote.get(domain);
+    if (!calls) throw new Error(`Not found calls for remote ${domain}`);
```

Comment:
make it return `string | undefined`

Reply:
https://github.com/nomad-xyz/nomad-monorepo/pull/79/commits/501bd29cee0dfdf0c5a341a18655b075b1f1b180